nautobot/core/forms/utils.py

```diff
@@ -35,7 +35,8 @@ def parse_numeric_range(string, base=10):
             begin, end = dash_range, dash_range
         begin, end = int(begin.strip(), base=base), int(end.strip(), base=base) + 1
         values.extend(range(begin, end))
-    return list(set(values))
+    # Remove duplicates and sort
+    return list(sorted(set(values)))
```

Comment:
Is the final cast to `list` needed? Doesn't `sorted` return a list?

Reply:
Sorted does seem to return a list, I will remove `list()`